test/built-ins/Math/pow/length.js

```diff
@@ -2,18 +2,16 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-es6id: 20.2.2.26
+esid: sec-math.pow
 description: >
   Math.pow.length is 2.
-info: >
-  Math.pow ( x, y )
```

Comment:
keep the info tag. 

the `description` tag is designed to be a single line, where info holds any extra information.